### PR TITLE
Feature/contribution guide

### PR DESCRIPTION
**Pull request by pengin7384**
Complete
- 기여 가이드 추가

### CODE REVIEW

**Comment by pengin7384** [comment]
@minsulee2 피드백 반영 푸시하였습니다

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Contribution Guidelines
+Ensure your pull request adheres to the following guidelines
+- Leave issues if you have any interesting ideas or bugs interesting ideas or bugs.
```
Comment (minsulee2) [inline contributing.md:3]: Duplicate words

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- Use English in all cases(Communication, Comment, Code review...)
+
+## Code contribution guide
+- Consist indentation of 4 spaces.
```
Comment (minsulee2) [inline contributing.md:7]: Without consist

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Contribution Guidelines
+Ensure your pull request adheres to the following guidelines
+- Leave issues if you have any interesting ideas or bugs interesting ideas or bugs.
+- Use English in all cases(Communication, Comment, Code review...)
```
Comment (minsulee2) [inline contributing.md:4]: , and Code review etc) would be better

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Code contribution guide
+- Consist indentation of 4 spaces.
+- File name uses snake case for file name.
```
Comment (minsulee2) [inline contributing.md:8]: Name a file as snake_case

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- Variable name uses pascal case for react components and camel case for others.
```
Comment (minsulee2) [inline contributing.md:9]: Please fix it

File: contributing.md (hunk 1 above)
Comment (pengin7384) [reply contributing.md:3]: Thanks Done!

File: contributing.md (hunk 2 above)
Comment (pengin7384) [reply contributing.md:7]: Done!

File: contributing.md (hunk 4 above)
Comment (pengin7384) [reply contributing.md:8]: `- Name a react component files as pascal case, others snake case.` Done!

File: contributing.md (hunk 3 above)
Comment (pengin7384) [reply contributing.md:4]: Done!

File: contributing.md (hunk 5 above)
Comment (pengin7384) [reply contributing.md:9]: `- Name a react component variable as pascal case, others camel case.` Done!

**Comment by pengin7384** [comment]
@minsulee2 피드백 반영 완료하였습니다. 파일네이밍 롤백한 부분도 적용하였습니다.

**Pull request merged**